src/main/scala-2.12/org/apache/james/gatling/jmap/scenari/realusage/JmapInboxHomeLoadingScenario.scala

```diff
@@ -1,4 +1,4 @@
-package org.apache.james.gatling.jmap.scenari
+package org.apache.james.gatling.jmap.scenari.realusage
```

Comment:
which one is unreal then?

Reply:
This scenario should just live in scenario, There's no such thing as "real" or not, just less complex ones and more complex ones.